src/jobs/pages/utvikler-i-oslo.md

```diff
@@ -1,67 +1,87 @@
+- Bonus. Du vil sammen med alle ansatte dele 30% av overskuddet per kvartal som en resultatbonus. Ansatte som er i foreldrepermisjon eller som er sykemeldte får også full bonus. Gjennomsnittlig bonus per år har siden 2019 vært på 71 079.
 - Gjennom Medeierskapsprogrammet i Variant [tilbys alle ansatte å kjøpe aksjer](https://blog.variant.no/invitasjon-til-%C3%A5-kj%C3%B8pe-aksjer-i-variant-as-27a29a307cb2) i Variant. Første april hvert år selger selskapet aksjer i selskapet til alle ansatte.
-- Vi har en [håndbok](https://handbook.variant.no/) for hva du enn måtte lure på. (Pro tip: sjekk ut [Andre goder og ytelser](https://handbook.variant.no/#andre-goder-og-ytelser))
+
+**_Alle våre vilkår ligger åpent_**
+
+- [Gode og rettferdige betingelser](https://handbook.variant.no/#Selve-livet)
+- [God lønn som er åpen for alle](https://www.variant.no/kalkulator)
+- [Innsikt og påvirkning](https://blog.variant.no/bli-en-bedre-variant-7e1926bdcfba#e27f) i alt som skjer
+- [Vokse faglig og hjelpe andre vokse](https://blog.variant.no/aapen-og-delt-kompetansebygging-c229771eee93)
+
+Vi har en [håndbok](https://handbook.variant.no/) for hva du enn måtte lure på. (Pro tip: sjekk ut "Andre goder og ytelser")
 
 ## Om Variant
 
 ![Christian, Jacob og Maciek hos Variant](/images/utvikler-jacob.png)
 Variant er menneskene som jobber her. Vi har kommet sammen for å være med å skape et konsept og en filosofi vi selv tror på. Hverdagen blir mer spennende av å ha et fellesskap som engasjeres av utvikling og hvordan det påvirker hverdagen. Forretningsmodell vår er enkel: Vi tilbyr flinke folk.
 
-Da er det helt selvsagt at vi må investere i kunnskap. Læreglede kommer i mange fasonger og vi prøver å tilrettelegge for at vi som varianter skal lære på den måten vi ønsker. Noen av oss driver podcasts ([her](http://bartjs.io/tag/podcast-episode/) og [her](https://kortslutning.fun/)), noen lager [kodevideoer](https://youtube.com/kodesnutt), noen underviser på NTNU. I tillegg samles vi hver måned til det vi [kaller en variantdag](https://blog.variant.no/tagged/variantdag); vår egen innedag for faglig påfyll. Og, ikke minst, for å møtes.
-
+Da er det helt selvsagt at vi må investere i kunnskap. Læreglede kommer i mange fasonger og vi prøver å tilrettelegge for at vi som varianter skal lære på den måten vi ønsker. Noen av oss driver podcasts ([her](http://bartjs.io/tag/podcast-episode/) og [her](https://kortslutning.fun/)), noen lager [kodevideoer](https://youtube.com/kodesnutt), noen underviser på NTNU. I tillegg samles vi hver måned til det vi [kaller en variantdag](https://blog.variant.no/tagged/variantdag); vår egen innedag for faglig påfyll. Og, ikke minst, for å møtes. I Oslo har også de ulike avdelingene sine egne <em>og noen kalde</em>. Eksempelvis <em>Kode og noen kalder</em>
+der vi jevnlig møtes for å utforske nye teknologier, vise frem noe fra prosjekt eller "mobprogrammere" med noe godt å drikke og spise 🔥
```

Comment:
Bra tillegg om KONK! Siden dette er for erfarne utviklere tror jeg det også hadde vært bra med en egen seksjon med moduler fra lederutvikling++ som starter opp nå førstkommende Felles Variantdag. Her kan sikkert Linn bistå med å skrive en tydeliggjørende tekst med essensen av det en vil oppnå i det programmet

Reply:
Godt forslag det. Jeg kan legge det til i neste iterasjon av stillingen sammen med nye metabilder etc. Tenker det er fint å få ut den oppdaterte teksten og de andre fiksene først 👍